.drone.yml

```diff
@@ -85,10 +98,19 @@ platform:
 
 steps:
   - name: katalog
-    image: quay.io/sighup/katalog:3.2.2_3
+    image: quay.io/sighup/mise:v2025.4.4
     pull: always
+    environment:
+      MISE_DATA_DIR: /mise-data
+      GITHUB_TOKEN:
+        from_secret: github_token
+    volumes:
+      - name: mise-cache
+        path: /mise-data
     depends_on: [ clone ]
     commands:
+      - eval "$(mise activate bash --shims)" && mise install  && mise use python@3.12
+      - python -m venv venv && . venv/bin/activate && pip install flake8 pytest pyyaml && export PATH="./venv/bin:$PATH" 
```

Comment:
Mmm.. I had no idea we were using pytest in this module. It may be a good idea to switch it to bats (very low prio, we can track this with an issue) to have it aligned with the rest of the modules.

In the meantime I'd use mise+uv that should be faster:
https://mise.jdx.dev/mise-cookbook/python.html#mise-uv